lessons/Algebra2/exploring-exponential-models/langs/en-us/pages/exploring-covid.adoc

```diff
@@ -0,0 +1,67 @@
+= Exploring the Covid Dataset
+++++
+<style>
+body.workbookpage td .autonum::after { content: ')' !important; }
+</style>
+++++
+For this page, you'll need to have the @starter-file{alg2-covid} open on your computer. If you haven't already, select **Save a Copy** from the "File" menu to make a copy of the file that's just for you.
+
+@n Click "Run", and evaluate `covid-table` in the Interactions Area. Do you know what each column means?
```

Comment:
I think it's confusing to ask questions that we don't actually want students to answer.
Also, I think it's a more useful to convention to put notes on column titles as comments in the starter file so that anyone who accesses the file has the information they need to make sense of the data.